holidays/countries/albania.py

```diff
@@ -41,7 +41,7 @@ def _add_with_observed(
                     + td(days=+2 if self._is_saturday(hol_date) else days)
                 ] = f"{hol_name} (Observed)"
 
-        super()._populate(year)
+        HolidayBase._populate(self, year)
```

Comment:
Actually it's because when it's monkey patched into the Holidays class, the method loses the reference as to what is considered `super()`. 

A stylistic cost of backwards compatibility!